packages/tailwindcss/tests/ui.spec.ts

```diff
@@ -160,6 +160,33 @@ test('composing shadow, inset shadow, ring, and inset ring', async ({ page }) =>
   )
 })
 
+test('shadow colors', async ({ page }) => {
+  let { getPropertyValue } = await render(
+    page,
+    html`<div id="x" class="shadow shadow-red-500"></div>
+      <div id="y" class="shadow-xl shadow-red-500"></div>`,
```

Comment:
That is some weird af formatting. Prettier seems to be okay with this too which feels better:

```suggestion
    html`
      <div id="x" class="shadow shadow-red-500"></div>
      <div id="y" class="shadow-xl shadow-red-500"></div>
    `,
```

Reply:
Nice, tweaked that too 👍